syntax_and_semantics/visibility.md

```diff
@@ -1,8 +1,8 @@
 # Visibility
 
-Methods are public by default: the compiler will always let you invoke them, even if there is no `public` keyword.
+Methods are public by default: the compiler will always let you invoke them. In fact, there isn't even a `public` keyword in the language!
```

Comment:
I prefered the previous sentence, which has a more neutral tone.
I suggest a simple and neutral: ``Methods are public by default: the compiler will always let you invoke them. There is no `public` keyword.``

Reply:
@j8r: Thanks for the feedback :)

You're correct about the surprised tone: I added it because it _is_ surprising that a language has `private` & `protected` _without_ also having `public` too.

Ruby, Python, Java, C, Objective-C, C++, C# & Swift all have `public` in their list of modifier syntax - and for many of those, `public` is the default. The only language I could trivially find that had `private` & `protected` without having `public` too was Scala.

I'm not saying it's better; I'm just saying it's _expected_ (which is why this issue arose for me in the first place!).

So I respectfully ask to leave that in.